includes/class-wc-stripe-order-helper.php

```diff
@@ -0,0 +1,613 @@
+<?php
+
+use Automattic\WooCommerce\Enums\OrderStatus;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * WC_Stripe_Order_Helper class.
+ */
+class WC_Stripe_Order_Helper {
+	/**
+	 * Meta key for Stripe currency.
+	 *
+	 * @string
+	 */
+	private const META_STRIPE_CURRENCY = '_stripe_currency';
+
+	/**
+	 * Meta key for Stripe fee.
+	 *
+	 * @string
+	 */
+	private const META_STRIPE_FEE = '_stripe_fee';
+
+	/**
+	 * Meta key for Stripe fee (legacy version).
+	 *
+	 * @string
+	 */
+	private const LEGACY_META_STRIPE_FEE = 'Stripe Fee';
+
+	/**
+	 * Meta key for Stripe net.
+	 *
+	 * @string
+	 */
+	private const META_STRIPE_NET = '_stripe_net';
+
+	/**
+	 * Meta key for Stripe net (legacy version).
+	 *
+	 * @string
+	 */
+	private const LEGACY_META_STRIPE_NET = 'Net Revenue From Stripe';
+
+	/**
+	 * Meta key for Stripe source ID.
+	 *
+	 * @string
+	 */
+	private const META_STRIPE_SOURCE_ID = '_stripe_source_id';
+
+	/**
+	 * Meta key for Stripe charge ID.
+	 *
+	 * @string
+	 */
+	private const META_STRIPE_CHARGE_ID = '_stripe_charge_id';
+
+	/**
+	 * Meta key for Stripe refund ID.
+	 *
+	 * @string
+	 */
+	private const META_STRIPE_REFUND_ID = '_stripe_refund_id';
+
+	/**
+	 * Meta key for Stripe intent ID.
+	 *
+	 * @string
+	 */
+	private const META_STRIPE_INTENT_ID = '_stripe_intent_id';
+
+	/**
+	 * Meta key for Stripe setup intent ID.
+	 */
+	private const META_STRIPE_SETUP_INTENT = '_stripe_setup_intent';
+
+	/**
+	 * Meta key for payment awaiting action.
+	 *
+	 * @string
+	 */
+	private const META_STRIPE_PAYMENT_AWAITING_ACTION = '_stripe_payment_awaiting_action';
+
+	/**
+	 * Gets the Stripe currency for order.
+	 *
+	 * @since 9.9.0
```

Comment:
All of the instances of `@since` should be `10.0.0`.

Reply:
Updated all of them in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4613/commits/0ffe7b37735109b0bf78924892610e0b87c1cd08